model.py

```diff
@@ -1,7 +1,6 @@
 import tensorflow as tf
+from utils import Screenshot
 
-IMG_W = 200
-IMG_H = 66
 OUT_SHAPE = 5
```

Comment:
Agreed. I thought about moving it too, but it obviously doesn't fit in the Screenshot class. I may take care of that later as I continue to play with things (unless you or someone else beats me to it).